optimade/validator/config.py

```diff
@@ -71,7 +71,7 @@
         "ENDS",
     ),
     DataType.TIMESTAMP: (
-        # "=" and "<=" are disabled due to issue with microseconds stored in database vs API response (see Materials-Consortia/optimade-python-tools/#606)
+        # N.B. "=" and "<=" are disabled due to issue with microseconds stored in database vs API response (see Materials-Consortia/optimade-python-tools/#606)
```

Comment:
```suggestion
        # N.B. "=" and "<=" are disabled due to issue with microseconds stored in database vs API response (see Materials-Consortia/optimade-python-tools#606)
```

Reply:
Neither work as links (even with appropriate url prefix) so I'm not particularly fussed